dv/cs_registers/README.md

```diff
@@ -8,19 +8,15 @@ It is a work in progress and only tests a handful of registers, and is missing m
 How to build and run the testbench
 ----------------------------------
 
-(from this directory)
-
 Verilator version:
 
    ```sh
-   make TOOL=verilator
-   ./build/lowrisc_ibex_tb_cs_registers_0/sim-verilator/Vtb_cs_registers
+     fusesoc --cores-root=. run --setup --build --run --target=sim lowrisc:ibex:tb_cs_registers
```

Comment:
nit: indent `fusesoc` with the start of the block (here and below)

You can remove the `--setup --build --run` part, it should be default if not supplied.

Possibly for consistency: add `--tool=verilator`?